SECP-service/src/main/resources/assets/app/views/portal/filter.html

```diff
@@ -0,0 +1,124 @@
+<body>
+<h1 class="portalheader"><i class="fa fa-filter" aria-hidden="true"></i> Filter</h1>
+
+<table id="filtertable" class="display" cellspacing="0" width="100%">
+    <button class="tableaddbutton" data-toggle="modal" data-target="#myModal">New Word <i class="fa fa-plus" aria-hidden="true"></i></button>
+    <thead>
+    <tr>
+        <th>Keyword</th>
+        <th>Permission Level(s)</th>
+        <th>Role(s)</th>
+        <th style="text-align: right">Action(s)</th>
+    </tr>
+    </thead>
+    <tfoot>
+    <tr>
+        <th>Keyword</th>
+        <th>Permission Level(s)</th>
+        <th>Role(s)</th>
+        <th style="text-align: right">Action(s)</th>
+    </tr>
+    </tfoot>
+    <tbody>
+    <tr>
+        <td>SECP</td>
```

Comment:
First rule of coding is DRY(Don't repeat yourself). I can see some repetition here. You can use ng-repeat to display this data I think. This would reduce our workload when we want to work on the javascript side. let me know if you need some help on how to go about this.

Reply:
@tosinoni will create a directive for the table. Once we have a database for filter table we will use `ng-repeat` to output all filter words.